zips/zip-0233.md

```diff
@@ -52,83 +49,86 @@ checks.
 
 # Abstract
 
-This ZIP proposes the introduction of a mechanism to voluntarily burn funds,
-removing those funds entirely from circulation on the network. This mechanism,
-in combination with ZIP 234 [^zip-0234] and ZIP 235 [^zip-0235], comprises a
-long-term strategy for the sustainability of the network. We will refer to the
-combined effects of these three ZIPs as the “Network Sustainability Mechanism”.
+This ZIP proposes the introduction of a mechanism to voluntarily remove funds
+entirely from circulation on the network. This mechanism, in combination with
+ZIP 234 [^zip-0234] and ZIP 235 [^zip-0235], comprises a long-term strategy for
+the sustainability of the network. We will refer to the combined effects of
+these three ZIPs as the “Network Sustainability Mechanism”.
 
 
 # Motivation
 
 This mechanism seeks to address concerns about the sustainability of the network
 design shared by Bitcoin-like systems:
 
-1. **Long Term Consensus Sustainability:** By enabling the burning of funds, the
-   network gains the ability to create "headroom" between the chain value and
-   $\mathsf{MAX\_MONEY}$. This lays necessary groundwork for extending the
-   block subsidy system, which currently has a clear final end date.
-2. **Benefits to ZEC Holders:** Burning funds reduces the supply of ZEC,
-   benefiting network users in proportion to their holdings without requiring
-   them to opt into any scheme, introducing extra risk, active oversight, or
-   accounting complexity.
+1. **Long Term Consensus Sustainability:** By enabling the removal of funds from
+   circulation, the network gains the ability to create "headroom" between the
+   chain value and $\mathsf{MAX\_MONEY}$. This lays necessary groundwork for
+   extending the block subsidy system, which currently has a clear final end
+   date.
+2. **Benefits to ZEC Holders:** Removing funds from circulation reduces the
+   supply of ZEC, benefiting network users in proportion to their holdings
```

Comment:
The rephrase throws into relief the problem that existed here before:
- Point 1 advocated for reducing issued / circulating supply.
- Point 2 advocated for reducing maximum supply (which is precisely what "burning funds reduces the supply" means in other chains).

Now that we have clarity about the intention of the NSM being to reduce the circulating supply, point 2 should be similarly reworded (and any claimed benefit adjusted appropriately):
```suggestion
   circulating supply of ZEC, benefiting network users in proportion to their holdings
```